examples/14_middleware/src/entries.tsx

```diff
@@ -13,7 +12,8 @@ export default defineEntries(
     const ctx = this.context as { count: number };
     ++ctx.count;
     return {
-      App: <App name={input || 'Waku'} count={ctx.count}  />,
+      App: <ServerContextWrapper context={ctx}><App name={input || 'Waku'} count={ctx.count} /></ServerContextWrapper>
```

Comment:
Yay, nice try!

Reply:
I was not able to set the React.cache in entries.tsx - only by using the wrapper the context worked.

Reply:
Yeah, I will set it higher in the tree when I implement it.